### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by Xottab-DUTY**

### CODE REVIEW

**Comment by CrossVR** [comment]
Is there any reason to keep the Mixed configuration? I think it's better to get rid of it.

**Comment by Xottab-DUTY** [comment]
Almost all suffixes removed. #78

I didn't remove only windows-specific ".dll", such as "d3d9.dll" and other

File: src/xrCore/xrCore.h [hunk 1]
```diff
@@ -102,7 +102,7 @@
 #endif
 
 #include <time.h>
-#define MODULE_NAME "xrCore.dll"
+#define MODULE_NAME "xrCore"
```
Comment (Kaffeine) [inline src/xrCore/xrCore.h]: @nitrocaster: is this safe?

Comment (nitrocaster) [inline src/xrCore/xrCore.h]: @Kaffeine It is. You could just read the [specification on MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684175%28v=vs.85%29.aspx).

Comment (Kaffeine) [inline src/xrCore/xrCore.h]: I already read about LoadModule(), as well as GetModuleHandle() and some other functions, but I dont know all MODULE_NAME usages. May be it's not just those methods argument. I have no IDE, nor the source code ATM on this pc.

Comment (nitrocaster) [inline src/xrCore/xrCore.h]: The only place it's used is xrCore.cpp:49:

``` cpp
GetModuleFileName(GetModuleHandle(MODULE_NAME), fn, sizeof(fn));
```

So actually `MODULE_NAME` should be either defined in xrCore.cpp or inlined on the line above.

**Comment by Kaffeine** [comment]
@nitrocaster 

> Need to check if it (hxGrid library) accepts file names without extensions

Would you test it? Or write a small instruction (for me or @Xottab-DUTY  or someone else) how to ensure that this works/doesn't work?
It seems to be used in Light module. Probably it would be enough to test light with the most capable Render (R4) in, say, local multiplayer game (just like in Armadas videos)?

**Comment by Xottab-DUTY** [comment]
I'll split this PR

**Pull request closed without merging**